README.md

```diff
@@ -2,56 +2,168 @@
+  - Conda (alternative, but containers preferred)
+- **Java**: Java 17 (or later, up to 24)
+- **Bash**: Bash 3.2 (or later)
+- **Memory**: Minimum 8GB RAM recommended
+- **Storage**: Sufficient disk space for input data, intermediate files, and outputs
+
+### Access Requirements
+
+- **PCGL Access Token**: Valid authentication token with submission permissions for your study
+- **Study Registration**: Your study must be registered in the PCGL system
+- **Participant Registration**: The participants in your submission batch must be registered in the PCGL system
+- **Network Access**: Connectivity to PCGL submission endpoints:
+  - File Manager service
+  - File Transfer service  
+  - Clinical submission service 
+
+### Input Data Requirements
+Please refer to **[Input Documentation](docs/input.md)** for the comprehensive parameter descriptions and file format specifications.
 
+- **Molecular Data Files**: 
+  - Supported formats: CRAM, BAM, VCF, BCF
+  - Files must include appropriate index files (e.g., .crai, .bai, .tbi)
+  - Files must be accessible from the specified `path_to_files_directory`
+
+- **Metadata Files**: 
+  - **Required**: `file_metadata.tsv`, `analysis_metadata.tsv`
+  - **Optional**: `workflow_metadata.tsv`, `read_group_metadata.tsv`, `experiment_metadata.tsv`, `specimen_metadata.tsv`, `sample_metadata.tsv`
+  - All metadata files must be in tab-separated (TSV) format
+  - Files must comply with the Custom data model of your study, which is the combination of PCGL Base and Extentions Data Model. For the latest version of the PCGL Base Data Model, please see the [latest release folder](https://drive.google.com/drive/u/1/folders/1vfNA7ajwh3WKkbVmswb6j9TuWKxaN9bB).
+
+### Environment Setup
+
+1. **Install Nextflow**:
+
+   ```bash
+   curl -s https://get.nextflow.io | bash
+   chmod +x nextflow
+   mv nextflow ~/bin/ # or add to your PATH
+   ```
 > [!NOTE]
-> If you are new to Nextflow and nf-core, please refer to [this page](https://nf-co.re/docs/usage/installation) on how to set-up Nextflow. Make sure to [test your setup](https://nf-co.re/docs/usage/introduction#how-to-run-a-pipeline) with `-profile test` before running the workflow on actual data.
+> If you are new to Nextflow, please refer to [this page](https://www.nextflow.io/docs/latest/install.html#installation) on how to set-up Nextflow. Make sure to test your setup before running the workflow on actual data.
 
-<!-- TODO nf-core: Describe the minimum required steps to execute the pipeline, e.g. how to prepare samplesheets.
-     Explain what rows and columns represent. For instance (please edit as appropriate):
+2. **Install Container Engine**:
+   - **Docker**: Follow [Docker installation guide](https://docs.docker.com/get-docker/)
+   - **Singularity**: Follow [Singularity installation guide](https://sylabs.io/guides/3.0/user-guide/installation.html)
+
+3. **Verify Installation**:
+   ```bash
+   nextflow info
+   docker --version  # or singularity --version
+   ```
```

Comment:
```
See section dependencies for required versions
```
Create link https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#section-links